io/net/net_data.py

```diff
@@ -158,7 +158,7 @@ def testinterface(self):
             self.fail("interface test failed")
         # up the interface
         process.system(if_up, shell=True)
-        time.sleep(4)
+        time.sleep(5)
```

Comment:
@hathyaga Are we sure this sleep is the best approach here ? Can't we check the interface state here with a timeout of 5 seconds ?